test/datalog_tests/function.fail.ast.expected

```diff
@@ -111,9 +111,16 @@ Only non-extern functions can be overloaded.
 extern function foo(x: u32): ()
 ^
 
-error: ./test/datalog_tests/function.fail.dl:8:21-8:39: expression '(ddlog_std::nth(components, 1))' must be of type 'Result<>', but its type is 'ddlog_std::Option<string>'
-    var last_name = components.nth(1)?;
-                    ^^^^^^^^^^^^^^^^^^
+error: ./test/datalog_tests/function.fail.dl:6:5-10:1: Type mismatch:
+expected type: ddlog_std::Result<(string,string),string>
+actual type: ddlog_std::Option<('a17,'a28)>
+in
+expression '{(var components = (/*ddlog_std::split, internment::split*/(name, " ")));
```

Comment:
I don't understand where this embedded comment comes from.
Are you showing a list of candidate functions that match?
The comment syntax is not the most appropriate.
Perhaps something like `One of [ddlog_std::split, internment::split]`